django_tables2/rows.py

```diff
@@ -205,24 +205,92 @@ def items(self):
             yield (column, self.get_cell(column.name))
 
 
+class BoundPinnedRow(BoundRow):
+
+    @property
+    def attrs(self):
+        '''
+        Return the attributes for a certain pinned row.
+        Add 'pinned-row' to css class attribute
+        '''
+        row_attrs = computed_values(self._table.pinned_row_attrs, self._record)
+        cssClass = "pinned-row"
+        cssClass = " ".join([
+            'even' if next(self._table._counter) % 2 == 0 else 'odd',
```

Comment:
This logic seems like it might be cleaner in a separate method (also used by `BoundRow.attrs`